.idea/gradvek.iml

```diff
@@ -2,10 +2,7 @@
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="true">
     <exclude-output />
-    <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$/springdb/src/main/java" isTestSource="false" />
-      <sourceFolder url="file://$MODULE_DIR$/springdb/src/test/java" isTestSource="true" />
-    </content>
+    <content url="file://$MODULE_DIR$" />
     <orderEntry type="inheritedJdk" />
```

Comment:
do we need this + the other file?

Reply:
Hmm.. I'm not sure how this file got there? LOL. I can delete if we are not using it. Perhaps, VSCode generated these extraneous file.

Reply:
yeah it looks like some kind of metadata file lol